app/views/handlers/summary.py

```diff
@@ -15,4 +17,20 @@ def get_context(self):
         block = self._schema.get_block(self._current_location.block_id)
         collapsible = block.get("collapsible", False)
         context = questionnaire_summary_context(collapsible)
+
+        submission_schema = self._schema.get_submission()
+
+        if submission_schema:
+            context["title"] = submission_schema.get("title")
+            context["submit_button"] = submission_schema.get("button")
+            context["guidance"] = submission_schema.get("guidance")
+            context["warning"] = submission_schema.get("warning")
```

Comment:
Are you simply merging the whole of the `submission_schema` Dict? If so, you could just merge the two...

```Python
context = {**context, **submission_schema}
```

Reply:
Not sure if this will work now after I've made the changes from Lewis' comments

Reply:
I think you are correct